ComponentKit/Core/Scope/CKComponentScope.h

```diff
@@ -49,11 +54,14 @@ class CKComponentScope {
   ~CKComponentScope();
 
   /** @return The current state for the component being built. */
-  id state() const;
+  id state(void) const;
+
+  /** @return A block that schedules a state update. Usually, use [CKComponent -updateState:mode:] instead. */
```

Comment:
It would be nice to explain why this exists. I'd also recommend explaining why using `-[CKComponent updateState:mode:]` is preferable.